.config/openbox/rc.xml

```diff
@@ -390,7 +390,7 @@
           <enabled>true</enabled>
```

Comment:
Maybe all `\<startupnotify\>` need to be `\<enabled\>false` in this section, but especially the popular terminal urxvt doesn't support them.